language/documentation/examples/diem-framework/move-packages/DPN/sources/DiemAccount.move

```diff
@@ -2335,7 +2335,7 @@ module DiemFramework::DiemAccount {
     spec module {} // switch documentation context back to module level
 
     /// # Access Control
-
+    ///
```

Comment:
The doc comment didn't require the consistency of the comments. I made some changes which the doc comments need to be consistent to be recognized. I got this idea because sometimes people would do comments like 
```
/// ------------------------------ line split mark -----------------------------------

/// doc comment content
fun a(){}
```

In this situation, the `line split mark` would take as doc comment with `doc comment content` which doesn't make too much sense to me.

I made the changes which the doc comment is only matched right before the language item.

Please let me know if you think this is not ok.

Reply:
A lot of comment changes in `diem-framework` are made because of it.